app/controllers/bananas2_controller.rb

```diff
@@ -72,7 +72,7 @@ def set_banana
     end
 
     # Never trust parameters from the scary internet, only allow the white list through.
-    def banana_params
+    def banana_params(k:)
```

Comment:
__[method_definition_validator]__
* Lint/DefinitionValidator: Keyword params is required. at [app/controllers/bananas2_controller.rb#L48](/actcat/sideci_rails_sandbox/blob/8f814ddecda9c71f1321383f85407d15ea95f368/app/controllers/bananas2_controller.rb#L48)